cmd/integration_test/build/buildspecs/build-eks-a-cli.yml

```diff
@@ -4,7 +4,23 @@ phases:
   build:
     commands:
     - make e2e
+    - |
+      for provider_test_file in test/e2e/*_test.go; do
+        provider=$(echo $(basename $provider_test_file) | cut -d_ -f1)
+        make e2e-tests-binary E2E_TAGS="e2e $provider" E2E_OUTPUT_FILE=bin/$provider/e2e.test
+      done
```

Comment:
nit: move to script? (can be a follow up)

Reply:
I'll do this in a follow-up PR along with other cleanups.